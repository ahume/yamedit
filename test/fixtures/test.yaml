test:
  yaml:
    # A comment
    structure: value # A comment
  # A comment
  other:
    structure: value # A comment
