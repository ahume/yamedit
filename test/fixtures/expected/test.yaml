test:
  yaml:
    # A comment
    structure: value # A comment
  # A comment
  other:
    structure: value2 # A comment
